apps/nsqadmin/main.go

```diff
@@ -9,7 +9,7 @@ import (
 	"syscall"
 
 	"github.com/BurntSushi/toml"
-	"github.com/judwhite/go-svc/svc"
+	svc "github.com/judwhite/go-svc"
```

Comment:
@jehiah I don't think you need the import alias. I'm not using it in the example project

https://github.com/judwhite/go-svc/blob/a2deda217eb3db0e1264c3386d5e41f8c8b08207/example/main.go#L10

Reply:
hmmm by convention i expect aliases when the package names doesn't match the directory name (i.e. `svc` and `go-svc`) but clearly there is more to `goimports` logic around this that I don't understand because it doesn't automatically add this alias (but leaves it if it's there). 

I think either approach here is ok 🤷‍♂️